docs/library/machine.Timer.rst

```diff
@@ -61,6 +61,9 @@ Methods
          frequency of the channel.
 
      - ``period`` - The timer period, in milliseconds.
+     
+     - ``freq`` - The timer freq, the upper bound of frequency is dependent on
```

Comment:
Would be good to mention the units, which are `Hz`.  Also use the full word "frequency" in the description.

Reply:
Yes, I agree with you.